packages/thrift-client/src/main/connections/request.ts

```diff
@@ -20,7 +20,9 @@ export class RequestConnection<TClient> extends HttpConnection<TClient> {
       // Encoding needs to be explicitly set to null or the response body will be a string
       encoding: null,
       headers: {
-        Connection: 'keep-alive',
+        'host': this.hostName,
```

Comment:
See my comment on the Axios connection. I don't think you want to set this to the host name (unless it is really the service name).